test/constant_time_test.c

```diff
@@ -221,56 +246,60 @@ static int test_sizeofs(void)
 static int test_binops(int i)
 {
     unsigned int a = test_values[i];
-    unsigned int g = test_values_s[i];
+    uint32_t a32 = test_values_32[i];
```

Comment:
test_values_32 has only 9 elements,
while test_values has 10.

Reply:
ADD_ALL_TESTS(test_binops, OSSL_NELEM(test_values));

Reply:
Yeah - I figured that out. New commit pushed.